### DISCUSSION TITLE
Create Adtran directory in module-types

### DISCUSSION THREAD

**Discussion by shatt79** [netbox-community/devicetype-library#921] [Q&A] [answered]
Can someone create a directory under module-types for Adtran?  I have a lot of modules to add.

**Comment by danner26** [comment, marked as answer]
Hello @shatt79, you can create the folder yourself as you start adding modules. Create a folder under module-types called `Adtran` and then start adding your modules
When you submit a PR, it will automatically add that folder and when merged, it will be added to the master branch

If you have any questions please let me know. Otherwise please mark this answered!
